atom-extension/package.json

```diff
@@ -12,7 +12,7 @@
     "node": "*"
   },
   "dependencies": {
-    "tree-sitter-usfm": "^1.0.0-alpha.1"
+    "tree-sitter-usfm3": "^3.0.0-alpha.1"
```

Comment:
Shouldn't this be updated to `.4`?

Reply:
.4 is for the python package. This is the setup for the atom module and is requiring the NPM module. The latest there is alpha.2 but 1 will also work.

BTW, this is the atom-plugin module, this is not yet functional.

Once we have got all the components working and in a proper share we can **keep same version number for all packages**. By that time we can also automate the packages publishing upon release. But we are not yet there. Still trying to get each component package in working condition.